### ISSUE TITLE
Separate Views for Map and List presentations

### ISSUE THREAD

**Issue by Sapeksha** [arjun27/covid-maps#312]
Currently, Maps and List presentations are both collectively shown in one view giving the user a feel for how both forms can be used to find the store that the user is interested in.

We should separate the Map and List presentations into their own views - which will help us achieve the following 2 things:
1. Give enough screen real estate to both views enabling us to build clean experiences in each
2. Observe the intent and preference of a user to use either View and giving us an indication of the value each of them offer

High definition mocks here at:
https://www.notion.so/Map-View-Design-d14d9c2163b6478483b974f9d7b3ea64

Design details:
1. Full screen views for Map and List
2. Enable flipping between views with buttons

**Comment by Sapeksha** [comment]
<img width="1300" alt="Screen Shot 2020-05-11 at 12 51 08 AM" src="https://user-images.githubusercontent.com/3829883/81537412-fd3c1b00-9321-11ea-95ea-67077f7013f8.png">

**Comment by Siddharth11** [comment]
@Sapeksha I think we should also add a label next to the list icon

**Comment by Sapeksha** [comment]
Yup that would be good @Siddharth11 Annu had the same comment as well
